dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/UnitOfWorkAwareProxyFactory.java

```diff
@@ -71,7 +71,7 @@ public UnitOfWorkAwareProxyFactory(HibernateBundle<?>... bundles) {
 
         try {
             final Proxy proxy = (Proxy) (constructorParamTypes.length == 0 ?
-                    factory.createClass().newInstance() :
+                    factory.createClass().getConstructor().newInstance() :
```

Comment:
This one is ok. I had to revert the Yaml one due to how it was being called. Per error prone, calling newInstance() without getConstructor() will be deprecated in JDK9 and is considered unsafe as it skips some exception handling.